src/bp/admin/ui/src/auth/Login.tsx

```diff
@@ -12,14 +12,21 @@ import { AuthMethodPicker } from './AuthMethodPicker'
 import LoginContainer from './LoginContainer'
 import { LoginForm } from './LoginForm'
 
-type RouterProps = RouteComponentProps<
+type  RouterProps = RouteComponentProps<
```

Comment:
Please add the ESLint and Prettier extensions on VS Code, this should have been fixed automatically when you save the file (it will automatically fix formatting)